types/cassanknex/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+	"extends": "dtslint/dt.json",
+  "rules": {
+    "no-unnecessary-generics": false,
```

Comment:
Please remove these rules and apply them locally for the line if absolutely needed

Reply:
Hey @sheetalkamat, can you explain how I'm supposed to write this typing and follow the `strict-export-declare-modifiers` rule?